.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,28 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ruby
+{
+  "name": "Ruby 2.5 on Debian",
+  "image": "ruby:2.5-slim-buster", // Specify the Ruby 2.5 image
+  "postCreateCommand": "RUBYGEMS_VERSION=3.3.27\ngem install rubygems-update -v ${RUBYGEMS_VERSION} > /dev/null 2>&1\nupdate_rubygems > /dev/null 2>&1\n", // Optional: Run this command after the container is created
```

Comment:
Do we need this?
Can we use already installed RubyGems?

BTW, can we run `bundle install` instead to install dependencies automatically?

Reply:
> can we use default RubyGems?

We could but IMO it is ideal to develop against the latest available to avoid potential bugs in gem / bundler.

> can we run bundle install?

Yes, I am somewhat indifferent there.  In this case I mostly wanted to make sure I was on latest available version of everything else so I had the best chance of any bugs being internal rather than external.

Reply:
Added `bundle install` in 5b7cd87